src/pipeline_func/clean.py

```diff
@@ -0,0 +1,266 @@
+               9: 'Sep', 10: 'Oct', 11: 'Nov', 12: 'Dec'}
+
+    df['month'] = df['month'].map(map_day)
+    df = df.drop(['datetime', 'period'], axis=1)
+    df['year'] = df['year'].apply(str)
+    return df
+
+#########################
+#     Run functions     #
+#########################
+
+
+def clean(new_dhis_path, old_dhis_path, new_dhis_report_path, old_dhis_report_path):
+
+    new_var_add_dict = get_variable_addition_dict(VAR_CORR, 'new')
+    old_var_add_dict = get_variable_addition_dict(VAR_CORR, 'old')
+
+    # Reporting data
+
+    new_dhis_report_df = get_reporting_data(new_dhis_report_path, "new")
+    make_note('new reporting data formatted for use')
+
+    old_dhis_report_df = get_reporting_data(old_dhis_report_path, "old")
+    make_note('old reporting data formatted for use')
+
+    # New instance
+
+    new_dhis_df = get_data(new_dhis_path, "new")
+
+    for x in list(new_var_add_dict.keys()):
+        new_dhis_df = compute_indicators(new_dhis_df, x, new_var_add_dict[x])
+    make_note('new data additional indicators created')
+
+    new_dhis_df = process_date(new_dhis_df)
+    make_note('new data transformed and cleaned')
+
+    # Old instance
+
+    old_dhis_df = get_data(old_dhis_path, "old")
+    for x in list(old_var_add_dict.keys()):
+        old_dhis_df = compute_indicators(old_dhis_df, x, old_var_add_dict[x])
+    make_note('old data additional indicators created')
+
+    old_dhis_df = process_date(old_dhis_df)
+    make_note('old data transformed and cleaned')
+
+    # Renaming variables
+
+    renaming_dict = dict(zip(VAR_CORR.name, VAR_CORR.identifier))
+
+    old_dhis_df['dataElement'].replace(renaming_dict, inplace=True)
+    new_dhis_df['dataElement'].replace(renaming_dict, inplace=True)
+    old_dhis_report_df['dataElement'].replace(renaming_dict, inplace=True)
+    new_dhis_report_df['dataElement'].replace(renaming_dict, inplace=True)
+    make_note('variable names updated')
+
+    # concatenate old and new instance
+
+    combined_df = pd.concat([old_dhis_df, new_dhis_df,
+                             old_dhis_report_df,
+                             new_dhis_report_df])
+    combined_df.reset_index(drop=True, inplace=True)
+    make_note('datasets concatenated')
+
+    # Dealing with duplicates dates
+
+    combined_df = combined_df.groupby(
+        ["dataElement", 'orgUnit', "year", "month"], as_index=False).agg({'value': 'sum'})
+    make_note('duplicate dates summed')
+
+    # Add district columns
+
+    districts = pd.merge(combined_df['orgUnit'], old_dhis_report_df[[
+        'orgUnit', 'districts']], how='left', left_on='orgUnit', right_on='orgUnit')
+    combined_df['districts'] = districts['districts']
+    make_note('district column added')
+
+    # Selecting only the facilities that are in both old and new
+    old_ids = set(old_dhis_report_df['orgUnit'].unique())
+    new_ids = set(new_dhis_report_df['orgUnit'].unique())
+    list_ids = list(new_ids.intersection(old_ids))
```

Comment:
now what you could do is just 

set(old_dhis_report_df['orgUnit'].tolist() + new_dhis_report_df['orgUnit'].tolist())

Reply:
you also could have filtered the dfs before concatting them together